hand_ranking_workflow.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+from typing import List
+from temporalio import workflow
+from hand_ranking import rank_hand
+from card import Card
```

Comment:
T: Consider running `black` and `isort` as we do elsewhere (looks like former may have run, but not latter)

Reply:
I haven't used either; links, please?

Reply:
https://github.com/psf/black and https://pycqa.github.io/isort/

Reply:
doing this now...

Reply:
Done